### PR TITLE
Mrc 3599 - Upgrade simple mailer

### PR DESCRIPTION
**Pull request by LekanAnni**
This PR updates simple mailer and changes implementation to utilise mailBuilder as library as been updated to use builder pattern.

Updated library now throws exception in staging and dev because config can't be located.

## Type of version change
_Delete as appropriate_

Major - Minor - Patches - None

## Checklist

- [x] I have incremented version number, or version needs no increment
- [x] The build passed successfully, or failed because of ADR tests

### CODE REVIEW

**Comment by EmmaLRussell** [comment]
@LekanAnni  What's the best way to test this?

**Comment by LekanAnni** [comment]
> @LekanAnni What's the best way to test this?

You should get an email. I have just tried resetting your password using your email. The test can be tried on staging server.

**Review by EmmaLRussell — APPROVED** [review]
This evidently works as I received the emails from the deployed version!

File: src/app/build.gradle [hunk 1]
```diff
@@ -52,7 +52,9 @@ dependencies {
     implementation 'org.postgresql:postgresql:42.3.3'
     implementation 'org.mindrot:jbcrypt:0.4'
     implementation 'com.github.spullara.mustache.java:compiler:0.9.5'
-    implementation "org.simplejavamail:simple-java-mail:4.4.5"
+    implementation "org.simplejavamail:simple-java-mail:6.5.0"
+    implementation 'jakarta.mail:jakarta.mail-api:1.6.7'
+    implementation 'com.sun.activation:jakarta.activation:2.0.1'
```
Comment (EmmaLRussell) [inline src/app/build.gradle:57]: Are these new dependences of javamail?

Comment (LekanAnni) [reply src/app/build.gradle:57]: Yes, java-mail now Jakarta-mail.

Comment (Nicolas-Dolan) [reply src/app/build.gradle:57]: Jakarta-mail - haha, sounds so dodgy!

**Review by Nicolas-Dolan — APPROVED** [review]
Just need to update the version again and can merge.

**Review by r-ash — APPROVED** [review]
Thanks for getting this sorted so quickly

**Pull request merged**